src/utils/strings.js

```diff
@@ -21,8 +21,10 @@ function snakeCaseToStartCase(str) {
  * @param {Array<number|string>} numbers - Array of numbers to format
  * @returns {string} Formatted string representation
  */
-function formatNumberSequence(numbers) {
+function formatNumberSequence(numbers, forFilename = false) {
```

Comment:
I think if you're going to pass in arguments, it should be custom delimiters, not a codepath based on a boolean.

Reply:
I've reverted this change.